icarusalg/Geometry/geometry_icarus.fcl

```diff
@@ -89,8 +89,8 @@ BEGIN_PROLOG
 
 icarus_geometry_template: {
   SurfaceY:        6.9e2    # in cm, vertical distance to the surface
-  Name:            "icarus_v2" # see ICARUS wiki for a list
```

Comment:
Is this right? Not sure if we want to be changing versions / gdml files.

Reply:
Agreed.  This is a weird change.  I'll revert it.